engine/crates/engine-v2/id-newtypes/src/range.rs

```diff
@@ -86,10 +86,10 @@ where
         let Some(first) = ids.next() else {
             return Some(Self::empty());
         };
-        let start = usize::from(*first);
+        let start: usize = (*first).into();
```

Comment:
I'm being a massive nitpicker but I really thought this was one place where `usize::from` is unambiguously better... 

(I prefer the clarity of `usize::from` in most of the places you've changed, but struggling to see any benefit to using into here since you have to specify the destination type anyway)

Reply:
I agree, did for `IdOperations` stuff when I copy pasted from your `regsitry-v2` code initially, but don't need it anymore. I'm going to revert those to like before. I agree `usize::from` is a lot better than `.into()`